go/ephemeral/team_ek.go

```diff
@@ -178,7 +186,8 @@ func boxTeamEKForUsers(ctx context.Context, g *libkb.GlobalContext, usersMetadat
 			RecipientGeneration: metadata.Generation,
 			Box:                 box,
 		}
-		boxes = append(boxes, boxMetadata)
+		boxes[i] = boxMetadata
+		i++
```

Comment:
If you want to do this efficient thing, without needing to introduce the new iteration variable, you can do it with the 3-argument form of `make` to set the capacity.